### ISSUE TITLE
Error on VimLeave

### ISSUE THREAD

**Issue by dapetcu21** [dhruvasagar/vim-prosession#81]
I get something like this every time I close nvim. Here's my init.vim: https://github.com/dapetcu21/vimrc

This is with just one buffer open (init.vim):

```
[coc.nvim] Error on autocmd VimLeavePre: request error nvim_command - Vim(mksession):E190: Cannot open "/Users/da_petcu21/.local/share/nvim/prosession//Users/da_petcu21/.config/nvim/init.vimUsers/Users/da_petcu21/.config/nvim/init.vimda_petcu21/Users/da_petcu21/.config/nvim/init.vimWork/Users/da_petcu21/.config/nvim/init.vimvscode-language-fuior.vim" for writing[coc.nvim] Error on autocmd VimLeavePre: request error nvim_command - Vim(mksession):E190: Cannot open "/Users/da_petcu21/.local/share/nvim/prosession//Users/da_petcu21/.config/nvim/init.vimUsers/Users/da_petcu21/.config/nvim/init.vimda_petcu21/Users/da_petcu21/.config/nvim/init.vimWork/Users/da_petcu21/.config/nvim/init.vimvscode-language-fuior.vim" for writing
```

**Comment by dhruvasagar** [comment]
@dapetcu21 It looks like an error stemming from fuior, I don't know much about it. Does not seem to be related to vim-prosession.

**Comment by dapetcu21** [comment]
How would that be stemming from fuior? What makes you think that? The only mention to fuior in that error is that vscode-language-fuior mention, which happens to be the dirname of a repo I've been working on recently (could as well have been part of the current dir).

Fuior is just a language I've been working on, btw. It has a tree-sitter grammar that I loaded with neovim's tree sitter support. If there's something wrong with it, I'd love to be able to debug it, though.

**Comment by dhruvasagar** [comment]
The error message you pasted :

```
Error on autocmd VimLeavePre: request error nvim_command - Vim(mksession):E190: Cannot open "...init.vimvscode-language-fuior.vim" for writing
```

**Comment by dhruvasagar** [comment]
@dapetcu21 Also the path mentioned there is weird and wrong, I don't know exactly where this is coming from, but vim-prosession is certainly not trying to write this file.

**Comment by dhruvasagar** [comment]
Let me know if this is still an issue and wasn't addressed in #86
